src/main/java/org/folio/event/service/UserTenantService.java

```diff
@@ -51,4 +76,35 @@ public Future<Boolean> deleteUserTenant(UserTenant userTenant, String tenantId,
     PostgresClient pgClient = pgClientFactory.apply(vertx, tenantId);
     return pgClient.withConn(conn -> tenantRepository.deleteUserTenant(conn, userTenant, tenantId));
   }
+
+  public Future<Void> isUsernameUpdatedAndUniqueAcrossTenants(User entity, User userFromStorage, Map<String, String> okapiHeaders, Conn conn, Context vertxContext) {
+    return ObjectUtils.notEqual(entity.getUsername(), userFromStorage.getUsername()) ? isUsernameUniqueAcrossTenants(entity, okapiHeaders, conn, vertxContext) : Future.succeededFuture();
+  }
+
+  public Future<Void> isUsernameUniqueAcrossTenants(User entity, Map<String, String> okapiHeaders, Conn conn, Context vertxContext) {
```

Comment:
Please add java doc and specify info that this check performing only when we are in consortium mode,
for common deployments we don't need to check cross tenant username uniqueness and always return succeded future

Reply:
The same java doc for getConsortiaCentralTenantId()
So if we are not in consortium mode - we always return null result and it is expected

Reply:
Added